src/torrent.cpp

```diff
@@ -9227,22 +9227,31 @@ namespace libtorrent
 
 	void torrent::queue_up()
 	{
+		// fix async position change calls for race conditions on changed torrents
+		if(!m_auto_managed || m_abort || is_finished()) return;
```

Comment:
I would guess that we tought that m_auto_managed is always true if queue management is enabled. 
We could have an issue if m_auto_managed is false on paused torrents.
Or we did worry that we stop the queue if 5 paused torrents are on top and only 3 queue entries could be active.